server_admin/topics/users/attributes.adoc

```diff
@@ -1,11 +1,18 @@
+[configuring-user-attributes]
```

Comment:
Originally, I planned to add the anchors, but if you add any, the anchor ID format is 
[id="noun_{context}"]  for a concept or a reference topic
OR
[id="gerund-noun_{context}"] for an assembly or a procedure.
So this should be [proc-configuring-user-attributes_{context}]
This format allows embedding assemblies into assemblies.